test/ssz_test.exs

```diff
@@ -1,17 +1,39 @@
 defmodule SSZTests do
   use ExUnit.Case
 
-  test "serialize and deserialize checkpoint object" do
+  def assert_roundtrip(type, value) do
+    {:ok, encoded} = LambdaEthereumConsensus.Ssz.to_ssz(type, value)
+    {:ok, decoded} = LambdaEthereumConsensus.Ssz.from_ssz(type, encoded)
+
+    assert decoded == value
+  end
+
+  test "serialize and deserialize checkpoint" do
     value = %{
       epoch: 12_345,
-      root:
-        <<1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-          0, 0, 1>>
+      root: Base.decode16!("0100000000000000000000000000000000000000000000000000000000000001")
     }
 
-    {:ok, encoded} = LambdaEthereumConsensus.Ssz.to_ssz(value)
-    {:ok, decoded} = LambdaEthereumConsensus.Ssz.from_ssz(encoded)
+    assert_roundtrip(Checkpoint, value)
```

Comment:
TIL: you can specify an atom like that, and that modules are atoms.

That being said, I don't think this is very idiomatic, it could be confusing. Is it difficult to be able to write an atom in lower case (i.e :fork_choice) that is then mapped to a pascal case struct in Rust?

Reply:
Ah that would be amazing!